### DISCUSSION TITLE
Developed 3D Globe Search Web App using your database - Thank You

### DISCUSSION THREAD

**Discussion by samsol38** [dr5hn/countries-states-cities-database#572] [Show and tell]
**Developed 3D Globe Search using your database - Thank You**

Hello Guys,

I am thankful to all contributors [@patrickdaze @fzzio @skrollme @bor0] to contribute this awesome Countries States Cities Database.

As a hobbyist project, I have developed 3D Globe Search using your database.

Repo: https://github.com/samsol38/3dglobesearch

Linkedin profile: https://linkedin.com/in/samir38

I mentioned your repo link in my web apps.

——————————
**Demo:**

**3D Globe Search Webapp:**

https://webapps-b7f67.firebaseapp.com

Demo Video: https://www.youtube.com/watch?v=gLhoaBanCnc
——————————
**Screenshot:**

**![3dglobesearch](https://user-images.githubusercontent.com/2776405/218939873-95bf315b-bcda-4321-88e0-9decaa216419.png)**

Thanks a lot once again.
SamSol

**Comment by dr5hn** [comment]
@samsol38 
That's great to hear! I'm glad to hear that you were able to use the database to create a 3D Globe Search.

If you have any questions or would like some more information about the database or any other related topics, please don't hesitate to ask!

**Comment by fzzio** [comment]
Thank you for your awesome work, @samsol38 💪🏽

**Comment by samsol38** [comment]
@fzzio @patrickdaze @skrollme @bor0 : 

Guys, Repo is live now about my 3D Globe Search App : https://github.com/samsol38/3dglobesearch

Thank you.

**Comment by yeahfine** [comment]
What I must tell you is that there is a serious error in your administrative division: Taiwan is a part of China and not an independent country!!! Similarly, the Hong Kong Special Administrative Region and the Macau Special Administrative Region are also part of China, and you cannot directly classify them as countries!!! Please correct this error immediately!!!
